activesupport/test/message_encryptor_test.rb

```diff
@@ -132,3 +140,35 @@ def munge(base64_string)
       ::Base64.strict_encode64(bits)
     end
 end
+
+class MessageEncryptorMetadataTest < ActiveSupport::TestCase
+  include SharedMessageMetadataTests
+
+  def setup
+    @secret    = SecureRandom.random_bytes(32)
+    @encryptor = ActiveSupport::MessageEncryptor.new(@secret, encryptor_options)
+
+    super
+  end
+
+  private
+    def generate(message, **options)
+      @encryptor.encrypt_and_sign(message, purpose: options[:purpose], expires_at: options[:expires_at], expires_in: options[:expires_in])
+    end
+
+    def parse(data, **options)
+      @encryptor.decrypt_and_verify(data, purpose: options[:purpose])
+    end
+
+    def encryptor_options
+      # serializer: default is Marshal
+      { cipher: "aes-256-gcm" }
```

Comment:
Why do we need the cipher? We've changed the default to be `aes-256-gcm`, right?

I still think we should go with:

```ruby
class MessageEncryptorMetadataMarshalTest < MessageEncryptorMetadataTest
  private
    def encryptor_options
      { serializer: Marshal } # Assain, just pass it in.
    end
end
```